2022-10-18 11:01:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "UUID" not found; SQL statement:
INSERT INTO "UserTable" ("UUID","name","age") VALUES ($1, $2, $3) [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.table.Table.getColumn(Table.java:749)
	at org.h2.command.Parser.parseColumn(Parser.java:1256)
	at org.h2.command.Parser.parseColumnList(Parser.java:1241)
	at org.h2.command.Parser.parseInsert(Parser.java:1641)
	at org.h2.command.Parser.parsePrepared(Parser.java:814)
	at org.h2.command.Parser.parse(Parser.java:689)
	at org.h2.command.Parser.parse(Parser.java:661)
	at org.h2.command.Parser.prepareCommand(Parser.java:569)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:92)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:288)
	at com.zaxxer.hikari.pool.ProxyConnection.prepareStatement(ProxyConnection.java:337)
	at com.zaxxer.hikari.pool.HikariProxyConnection.prepareStatement(HikariProxyConnection.java)
	at io.getquill.context.jdbc.JdbcContextVerbExecute.$anonfun$executeAction$1(JdbcContextVerbExecute.scala:35)
	at io.getquill.context.jdbc.JdbcContextVerbExecute.$anonfun$executeAction$1$adapted(JdbcContextVerbExecute.scala:34)
	at io.getquill.context.qzio.ZioJdbcUnderlyingContext.$anonfun$withConnectionWrapped$3(ZioJdbcUnderlyingContext.scala:68)
	at zio.ZIOCompanionVersionSpecific.$anonfun$attempt$1(ZIOCompanionVersionSpecific.scala:105)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:965)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:887)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:818)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:929)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:887)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:818)
	at zio.internal.FiberRuntime.evaluateEffect(FiberRuntime.scala:330)
	at zio.internal.FiberRuntime.evaluateMessageWhileSuspended(FiberRuntime.scala:448)
	at zio.internal.FiberRuntime.drainQueueOnCurrentThread(FiberRuntime.scala:223)
	at zio.internal.FiberRuntime.run(FiberRuntime.scala:140)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
2022-10-18 11:01:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Column "T.UUID" not found; SQL statement:
SELECT t."name" AS name, t."age" AS age FROM "UserTable" t WHERE t."UUID" = $1 [42122-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:502)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:244)
	at org.h2.expression.ExpressionColumn.optimizeOther(ExpressionColumn.java:226)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:213)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:139)
	at org.h2.expression.Expression.optimizeCondition(Expression.java:148)
	at org.h2.command.query.Select.prepareExpressions(Select.java:1177)
	at org.h2.command.query.Query.prepare(Query.java:218)
	at org.h2.command.Parser.prepareCommand(Parser.java:575)
	at org.h2.engine.SessionLocal.prepareLocal(SessionLocal.java:631)
	at org.h2.engine.SessionLocal.prepareCommand(SessionLocal.java:554)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1116)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:92)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:288)
	at com.zaxxer.hikari.pool.ProxyConnection.prepareStatement(ProxyConnection.java:337)
	at com.zaxxer.hikari.pool.HikariProxyConnection.prepareStatement(HikariProxyConnection.java)
	at io.getquill.context.jdbc.JdbcContextVerbExecute.$anonfun$executeQuery$1(JdbcContextVerbExecute.scala:42)
	at io.getquill.context.qzio.ZioJdbcUnderlyingContext.$anonfun$withConnectionWrapped$3(ZioJdbcUnderlyingContext.scala:68)
	at zio.ZIOCompanionVersionSpecific.$anonfun$attempt$1(ZIOCompanionVersionSpecific.scala:105)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:965)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:887)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:818)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:929)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:887)
	at zio.internal.FiberRuntime.runLoop(FiberRuntime.scala:818)
	at zio.internal.FiberRuntime.evaluateEffect(FiberRuntime.scala:330)
	at zio.internal.FiberRuntime.evaluateMessageWhileSuspended(FiberRuntime.scala:448)
	at zio.internal.FiberRuntime.drainQueueOnCurrentThread(FiberRuntime.scala:223)
	at zio.internal.FiberRuntime.run(FiberRuntime.scala:140)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
